File: InJava/app/src/main/java/com/barryzeha/quizquestion/model/database/RankingDB.java
package com.barryzeha.quizquestion.model.database;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import com.barryzeha.quizquestion.model.RankingModel;

/****
 * Project QuizQuestion
 * Created by Barry Zea H. on 01/10/2022
 * Copyright (c)  All rights reserved.
 ***/
@Database(entities = {RankingModel.class}, version=1)
public abstract class RankingDB extends RoomDatabase {
    public abstract RankingDao rankingDao();
}

File: InKotlin/app/src/main/java/com/barryzeha/quizquestion/model/database/RankingDB.kt
package com.barryzeha.quizquestion.model.database

import androidx.room.*
import androidx.room.RoomDatabase
import com.barryzeha.quizquestion.model.RankingModel

/****
 * Project QuizQuestion
 * Created by Barry Zea H. on 27/09/2022
 * Copyright (c)  All rights reserved.
 ***/
@Database(entities = [RankingModel::class], version = 1)
abstract class RankingDB :RoomDatabase(){
    abstract fun rankingDAO(): RankingDAO
}